client/src/components/small-display/SmallDisplayFileContextMenu.vue

```diff
@@ -66,15 +80,15 @@
           <ion-item
             button
             v-show="!multipleFiles && isFile && isDesktop()"
-            @click="onClick(FileAction.Open)"
+            @click="onClick(FileAction.Preview)"
```

Comment:
Revert this change, this is the `Open` button on desktop to open files in the explorer.